### PR TITLE
fix(fxa-content-server): validate log in fxa-content-server.js

### PR DESCRIPTION
**Pull request by sunakshi96**
Ref #4841

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 0d160b3bf55cb3827c8bfa2a0c31bf6a96749d6a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: server/bin/fxa-content-server.js [hunk 1]
```diff
@@ -156,9 +156,8 @@ function makeApp() {
   // log any joi validation errors
   app.use((err, req, res, next) => {
     if (err && err.isJoi) {
-      logger.error({
+      logger.error('validation.error',{
```
Comment (vladikoff) [inline server/bin/fxa-content-server.js]: @sunakshi96 need a style fix, there should be a space between `,{`, such that `, {`

**Comment by vladikoff** [comment]
@sunakshi96 you fixed the title but the commit message (`fix validation log in fxa-content-server.js`) still needs to be fixed. Example how to do that: https://www.git-tower.com/learn/git/faq/edit-fix-commit-message

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 0d160b3bf55cb3827c8bfa2a0c31bf6a96749d6a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 27009b6379d90828feb2c997127dceacabcc9679
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: server/bin/fxa-content-server.js (hunk 1 above)
Comment (sunakshi96) [inline server/bin/fxa-content-server.js]: alright

**Comment by vladikoff** [comment]
@sunakshi96 thanks for the updates, let's wait for the CI checks to make sure no tests broke by this change

**Pull request closed without merging**